api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -81,24 +88,24 @@ public WorkspacesApi workspacesApi(@Qualifier(END_USER_API_CLIENT) ApiClient api
 
   @Bean
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
-  public BillingApi billingApi(@Qualifier(ALL_OF_US_API_CLIENT) ApiClient apiClient) {
+  public BillingApi billingApi(@Qualifier(SERVICE_ACCOUNT_API_CLIENT) ApiClient apiClient) {
     // Billing calls are made by the AllOfUs service account, rather than using the end user's
     // credentials.
     BillingApi api = new BillingApi();
     api.setApiClient(apiClient);
     return api;
   }
 
-  @Bean(name = ALL_OF_US_GROUPS_API)
+  @Bean(name= SERVICE_ACCOUNT_GROUPS_API)
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
-  public GroupsApi groupsApi(@Qualifier(ALL_OF_US_API_CLIENT) ApiClient apiClient) {
+  public GroupsApi groupsApi(@Qualifier(SERVICE_ACCOUNT_API_CLIENT) ApiClient apiClient) {
     // Group/Auth Domain creation and addition are made by the AllOfUs service account
     GroupsApi api = new GroupsApi();
     api.setApiClient(apiClient);
     return api;
   }
 
-  @Bean(name = END_USER_GROUPS_API)
+  @Bean(name=END_USER_GROUPS_API)
```

Comment:
Nit: Can we do consistent spacing here? We have no spaces here, and above have a single space after the `=` sign. I don't have any strong feelings what we should do, just as long as we are consistent.

Reply:
Good catch, done. I'm 100% for consistency, I just haven't set up IntelliJ perfectly yet. I'm hoping we can get some pre-commit hooks that relieve us all from the mental burden of remembering code-formatting rules like this.

Reply:
If you do figure that out, it would be great to add that documentation to the handbook.